test/integration/mesos/mesos_helpers.bash

```diff
@@ -2,11 +2,36 @@
 
 load ../helpers
 
-MESOS_CLUSTER_ENTRYPOINT=${MESOS_CLUSTER_ENTRYPOINT:-0.0.0.0:5050}
+MESOS_IMAGE=jimenez/mesos-dev:clang
+MESOS_MASTER_PORT=$(( ( RANDOM % 1000 )  + 7000 ))
+BASE_PORT=2375
+
+# Start mesos master and slave.
+function start_mesos() {
+	MESOS_MASTER=$(
+	docker_host run -d --name mesos-master --net=host \
+	$MESOS_IMAGE /mesos/build/bin/mesos-master.sh --ip=127.0.0.1 --work_dir=/ --registry=in_memory --port=$MESOS_MASTER_PORT
+		)
+
+	MESOS_SLAVE=$(
+	docker_host run -d --name mesos-slave --volumes-from node-0 -e DOCKER_HOST="${HOSTS[0]}" -v /sys/fs/cgroup:/sys/fs/cgroup --net=host \
+	$MESOS_IMAGE /mesos/build/bin/mesos-slave.sh --master=127.0.0.1:$MESOS_MASTER_PORT --containerizers=docker --hostname=127.0.0.1 --port=$(($MESOS_MASTER_PORT + 1))
```

Comment:
I don't think it will work, the mesos-slave can't access the image, here is the current setup:

```
jenkins 
 └ docker (here the image jimenez/mesos-dev:clang is pulled)
   └ docker (there the image is not)
   └ mesos-slave 
   └ mesos-master
```